assignments/functionTypes.js

```diff
@@ -91,53 +207,157 @@ convertToString(21); // "21"
  * @param {number} y
  * @return {number} the result
  */
+    //Function declaration:
+    function calc(operation, x, y){
+      if(operation === "+"){
+        return x + y;
+      } else if (operation==="-"){
+        return x-y;
+      } else if (operation==="*"){
+        return x*y;
+      } else (operation==="/")
+        return x/y;
+    }
 
 /**
  * Returns true if `a` is greater than `b`.
  * @param {number} a
  * @param {number} b
  * @return {boolean} `a` is larger than `b`
- */
-
+ */ 
+//function declation:
+function larger(a,b){
+  return Boolean(a>b);
+}
+//function expression:
+let larger=function (a,b){
+  return Boolean(a>b);
+}
+//arrow function:
+let larger=(a,b)=> Boolean(a>b);
+//Arrow function with curley Bracket:
+let larger=(a,b) =>{
+  return Boolean(a,b);
+}
+ 
 /**
  * Returns true if `a` is less than `b`.
  * @param {number} a
  * @param {number} b
  * @return {boolean} `a` is smaller than `b`
  */
+//function declation:
+function smaller(a,b){
+  return Boolean(a<b);
+}
+//function expression:
+let smaller=function (a,b){
+  return Boolean(a<b);
+}
+//arrow function:
+let smaller=(a,b)=> Boolean(a<b);
+//Arrow function with curley Bracket:
+let smaller=(a,b) =>{
+  return Boolean(a<b);
+}
 
 /**
  * Returns true if `a` and `b` are equal.
  * @param {number} a
  * @param {number} b
  * @return {boolean} the numbers are equal
  */
+//function declation:
+function equals(a,b){
+  return Boolean(a=b);
+}
+//function expression:
+let equals=function (a,b){
+  return Boolean(a=b);
+}
+//arrow function:
+let equals=(a,b)=> Boolean(a=b);
+//Arrow function with curley Bracket:
+let equals=(a,b) =>{
+  return Boolean(a=b);
+}
 
 /**
  * Returns the smallest value of two numbers.
  * @param {number} x
  * @param {number} y
  * @return {number} the smallest number
  */
-
+  function smallest(a,b) {
+  var c = Math.min(a, b);
+  return c;
+  }
+ 
 /**
  * Returns the largest value of two numbers.
  * @param {number} x
  * @param {number} y
  * @return {number} the largest number
  */
+//Function Declaration:
+    function largest(a,b) {
+      var c = Math.max(a, b);
+      return c;
+      }
+    
+// Function expression:
+let largest= function(a,b) {
+  var c = Math.max(a, b);
+      return c;
+}
+//Arrow function with curley bracket:
+let largest = (a,b) => {
+  var c = Math.max(a, b);
+      return c;
+}
+
 
 /**
  * Returns true if `n` is even.
  * @param {number} n
  * @return {boolean} the number is even
  */
+      //Function declation:
+
+      function even(n){
+        return Boolean(n%2==0);
```

Comment:
Again `Boolean(n%2==0)` `==` always returns a boolean value you need not convert explicitly.